Add power usage summary page

airimovici opened the pull request.

airimovici commented:
https://github.com/mozilla-frontend-infra/firefox-health-dashboard/issues/630

davehunt commented:
As discussed, we're going to adjust this to have a section for each test (idle, idle-background, speedometer). Within each section we'll have a graph for each device (Moto G5, Pixel 2). These graphs will show results for all targets (GeckoView Example, Firefox ESR, Firefox Preview). This will allow for easy comparison between our applications on each device.

airimovici commented:
I did the changes required for another issues in this PR:

I changed the name from "Fennec68" to "Firefox ESR": https://github.com/mozilla-frontend-infra/firefox-health-dashboard/issues/633

I removed the Pixel 2 (arm7) from graphs: https://github.com/mozilla-frontend-infra/firefox-health-dashboard/issues/638

Also the issue "Show browsers in single chart for power tests" is also done here: https://github.com/mozilla-frontend-infra/firefox-health-dashboard/issues/622

klahnakoski commented:
@airimovici  is this ready for review and merge?

airimovici commented:
@klahnakoski  It should be ready, but after the last cleanup commit the CI is failing...

airimovici commented:
@klahnakoski Can you please take a new look at this PR?

klahnakoski commented on src/android/index.jsx:
```diff
@@ -243,24 +243,24 @@ class Android extends Component {
+                  //       },
+                  //     ],
+                  //   },
+                  // },
```
I assume you are keeping this config for later

klahnakoski replied:
...if not, then remove them

klahnakoski commented on src/power/power.jsx:
```diff
@@ -0,0 +1,46 @@
+    const timeDomain = new TimeDomain({ past: '2month', interval: 'day' });
+    const suites = selectFrom(COMBOS)
+      .groupBy('suiteLabel')
+      .map(([v]) => ({ suiteId: v.suite, suiteLabel: v.suiteLabel })).toArray();
```
You do not need to change `suite` to `suiteId`

Instead of 

      const suites = selectFrom(COMBOS)
      .groupBy('suiteLabel')
      .map(([v]) => ({ suiteId: v.suite, suiteLabel: v.suiteLabel })).toArray();

use

      const suites = selectFrom(COMBOS)
      .groupBy('suiteLabel')
      .map(first)
      .toArray();

and change the `suiteId` to `suite`
